### DISCUSSION TITLE
Can the viewer be docked?

### DISCUSSION THREAD

**Discussion by bpan2020** [doublecmd/doublecmd#2008] [General]
Can the viewer be docked as a window or tab in an explorer panel? Can we also configure it to automatically preview selected files (no F3 keypress needed)?

**Comment by alexx2000** [comment]
[Quick View Panel](https://doublecmd.github.io/doc/en/help.html#mnu_view)
[Quick view](https://doublecmd.github.io/doc/en/viewer.html#quick)

**Comment by bpan2020** [comment]
Great, thank you! It seems that the 'Quick View Panel' is docked in one of the two file panels. However, it gets disabled when switching any panel (active or inactive) to another tab. How can I disable this feature and keep it enabled when switching any panel (active or inactive) to another tab?

**Reply by alexx2000** [reply]
No way.

**Reply by bpan2020** [reply]
Could we consider adding this feature? It might significantly improve previewing efficiency when dealing with a large number of files in different tabs.

**Comment by bpan2020** [comment]
And can the image be stretched to fit the window size when previewing in this mode?

**Reply by antodc** [reply]
In the bar below the image you should see "preview". Right-click on it -> image -> stretch.

**Reply by bpan2020** [reply]
Thank you antodc. It works.
> In the bar below the image you should see "preview". Right-click on it -> image -> stretch.
